constants/users.js

```diff
@@ -1,2 +1,8 @@
 const userStatusEnum = ['ooo', 'idle', 'active']
-module.exports = userStatusEnum
+const ROLES = {
+  APPOWNER: 'app_owner',
+  DEFAULT: 'default',
+  MEMBER: 'member',
+  SUPERUSER: 'super_user'
+}
```

Comment:
Are we going to have `admin` role?

Reply:
These were the available roles that I had added
If we have documented about having admin role
I will add it accordingly @swarajpure